guides/elevation.md

```diff
@@ -47,14 +44,23 @@ In order to use the mixin, you must import `~@angular/material`:
 }
 ```
 
-For convenience, you can use the `elevation-transition` mixin to add a transition when the
-elevation changes:
+### Overridable elevation
+
+When authoring a component, you may want to specify a default elevation that the component consumer
+can override. You can accomplish this by using the `overridable-elevation` Sass mixin. This behaves
```

Comment:
[stretch] adding an example of the override to show this overridable usecase?

I'm not strongly opinionated on this

Reply:
I'll pass on this since I think it's kind of niche and also laziness.